doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3823,6 +3823,19 @@ Trusted Firmware-M (TF-M)
 
 The issues in this section are related to the TF-M implementation in the |NCS|.
 
+.. rst-class:: v2-5-0
+
+NCSDK-24804: TF-M does not withouth gpio0 node enabled in devicetree
```

Comment:
```suggestion
NCSDK-24804: TF-M does not build when the gpio0 node is disabled in devicetree
```

Reply:
Also, should we write GPIO0?

Reply:
I was using lower-case because I'm referring to the node in devicetree, which is lower-case.

Reply:
I just noticed that we are writing GPIO0 and GPIO1 in Kconfig file as well, so I thought we should align :)